client-app/src/app/activity-form/Activity-form.component.ts

```diff
@@ -4,8 +4,8 @@ import { Router } from '@angular/router';
 
 @Component({
   selector: 'app-activity-form',
-  templateUrl: './activity-form.component.html',
-  styleUrls: ['./activity-form.component.css']
+  templateUrl: './Activity-form.component.html',
+  styleUrls: ['./Activity-form.component.css']
```

Comment:
Why did you change these back to capital?

Reply:
The build on travis gave errors when it was lower case,  All four files within activity form were uppercase.  Without this the build failed.  We could go the other way and make every instance where those files are called lower case, this was just the quick fix.

Reply:
Info-form.components also had similar inconsitencies.
So I just made them consistent.

Reply:
I know the build works locally, but travis was spitting this error.  
ERROR in : Couldn't resolve resource ./activity-form.component.css relative to /home/travis/build/Rusty89/SeniorCenterManagmentStudio/client-app/src/app/activity-form/Activity-form.component.ts

Reply:
@RialJohnson

Reply:
I'd rather fix them to be all lowercase and maintain this consistency in the future.